worth2/templates/base.html

```diff
@@ -55,27 +58,31 @@
     </ul>
 
     <ul class="nav navbar-nav navbar-right">
-        {% if request.user.profile.is_participant %}
+        {% if user.profile.is_participant %}
```

Comment:
good catch - I need to remove that. I thought it was okay to put anonymous checks like that in the templates at first, but it ends up getting really confusing. Anonymous users should just always get redirected to the login page in WORTH.